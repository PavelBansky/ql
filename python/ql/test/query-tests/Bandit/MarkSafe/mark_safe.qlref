Bandit/mark_safe.ql