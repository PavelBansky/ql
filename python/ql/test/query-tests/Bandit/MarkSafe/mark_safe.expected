| Test.py:4:9:4:35 | Attribute() | Use of mark_safe() may expose cross-site scripting vulnerabilities |
